R/article.R

```diff
@@ -556,10 +556,69 @@ springer_article <- function(..., keep_tex = TRUE,  citation_package = "natbib",
 #' \samp{https://www.tandf.co.uk/journals/authors/InteractCADLaTeX.zip}.
 #' @export
 #' @rdname article
-tf_article <- function(..., keep_tex = TRUE, citation_package = "natbib") {
+tf_article <- function(..., keep_tex = TRUE, citation_package = "natbib",
+                       biblio_style = "CAD", pandoc_args = NULL) {
+  styles <- list(
+    APA = list(
+      bst = "apacite",
+      cmd = "\\usepackage[natbibapa]{apacite}
+\\setlength\\bibhang{12pt}
+\\renewcommand\\bibliographytypesize{\\fontsize{10}{12}\\selectfont}"
```

Comment:
Would that be easier maintenance to store a character vector of line and then join them with new line in `pandoc_variable_arg()` call later ? 

I don't want this to break in the future because of accidental reformatting or else.